api/boards.php

```diff
@@ -0,0 +1,37 @@
+<?php
+$method = $_SERVER['REQUEST_METHOD'];
+$params = $_GET;
+$payload = file_get_contents('php://input');
+$filename = '../data/' . $params["id"] . '.json';
+var_dump([
+  'method' => $method,
+  'params' => $params,
+  'payload' => $payload,
+  'filename' => $filename,
+]);
+if($method  == "POST") {
```

Comment:
There is a space too much. It is also recommended to do a strict comparison (`===` instead of `==`), as PHP is not a type safe language. On another note, this would also be the perfect use case for using a `switch` statement instead of multiple `elseif`s, but I would not enforce this.